### ISSUE TITLE
_lookup() got multiple values for argument 'name' for Ansible versions < ansible-2.14

### ISSUE THREAD

**Issue by gilesw** [dcostakos/doppler#2]
Testing out doppler and I got this error. Shame as the software looks great. Could it be the old version of Ansible I'm using?

```
fatal: [xxxx]: FAILED! => {"msg": "An unhandled exception occurred while templating '{{ lookup('dcostakos.doppler.doppler_secrets', project='x', token='dp.st.dev.Dxxxx', config='dev', name='XXX') }}'. Error was a <class 'ansible.errors.AnsibleError'>, original message: Unexpected templating type error occurred on ({{ lookup('dcostakos.doppler.doppler_secrets', project='registry', token='dp.st.dev.DkjMxxxx', config='dev', name='XXX') }}): _lookup() got multiple values for argument 'name'"}
```

```
ansible 2.9.9
  python version = 3.8.10 (default, Mar 31 2023, 12:16:43) [Clang 12.0.0 (clang-1200.0.32.29)]
```

**Comment by gilesw** [comment]
```
pip list | grep requests
requests                                        2.31.0
requestsexceptions                              1.4.0
```

**Comment by gilesw** [comment]
I've hit this with another lookup module, I assume that name is a reserved word somehow in older ansible versions

**Comment by dcostakos** [comment]
Giles!  Thanks for opening this issue!  I'm honestly just excited that anyone is using this collection and I'm happy to help on it.

What is the output of "ansible --version" in your system?

And, please note this issue in upstream Ansible which is likely related to yours: https://github.com/ansible/ansible/issues/79838

**Comment by gilesw** [comment]
ansible 2.9.9, ah cool nice one for finding it

**Comment by dcostakos** [comment]
Sorry Giles, I got sick!  But I'm back and starting to test.  Just as an FYI, Ansible 2.9 is definitely not being developed any longer.  You didn't share which version of python you're using, so I hope it's not python 2.7 or something.  I grabbed the oldest active verison 3 python I could for my testing.   Does this environment look acceptable to you? 

```
:~/ansible/old_python] $ ansible --version
ansible 2.9.9
  config file = None
  configured module search path = ['/Users/dcostakos/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /Users/dcostakos/.pyenv/versions/3.8.18/lib/python3.8/site-packages/ansible
  executable location = /Users/dcostakos/.pyenv/versions/3.8.18/bin/ansible
  python version = 3.8.18 (default, Jan 16 2024, 08:21:07) [Clang 15.0.0 (clang-1500.1.0.2.5)]
```

**Comment by dcostakos** [comment]
Well I'm assuming my environment has is okay.  I've completed some testing and will release a 1.0.2 version of this for testing today.

**Comment by dcostakos** [comment]
Giles, I hope this is fixed with the stable-1.0.2 branch.  If you get some time, try to do this to try out the new version:

`ansible-galaxy collection install git+https://github.com/dcostakos/doppler.git,stable-1.0.2`

Let me know how it goes and if it works I'll post the fix to galaxy.

Now, I should note that really the oldest version of Ansible upstream is supporting is 2.14 and if you were to upgrade to that version (or newer) this problem would not occur.  Happy to fix it tough because I'm jus tglad someone is using this.

**Comment by dcostakos** [comment]
I haven't heard from you, so I plan to close this as fixed with teh above note.  I will not publish this new content to galaxy however.  If you'd like me to consider that, open a new issue.  Cheers and thanks.
